packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -1543,6 +1769,9 @@ class DiagnosticsProperty<T> extends DiagnosticsNode {
   /// generating the string description.
   final String tooltip;
 
+  /// Whether a value of null should trigger a warning.
```

Comment:
these docs talk should talk about the implications on `level`

Reply:
done